js/index.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(function() {
+
+console.log('at doc ready');
+
+	$('.readless a').hide();
+	$('.readmore').click(articleOpen);
+	$('.readless').click(articleClose);
+	$('.learnmore').click(learnMoreOpen);
+
+
+	// article read more and read less
+	function articleOpen() {
+		$('#show-this-on-click').slideDown();
+		$('.readmore a').hide();
+		$('.readless a').show();
+	}
```

Comment:
You left out the `event.preventDefault` which prevents the `a` link element from it's normal behavior of reloading the page. Without this, clicking on each link will just reload the JS and prevent your JS from running.